core/src/components/text-area/text-area.tsx

```diff
@@ -28,16 +28,31 @@ const onChange = (props: TextAreaProps) => (e: ChangeEvent) => {
 
 export interface TextAreaProps {
 	// Uncontrolled
+	/**
+	 * Initial value of the input
+	 */
 	defaultValue?: string;
 	forwardedRef?: ForwardedRef<HTMLTextAreaElement>;
 	// Controlled
 	value?: string;
+
+	/**
+	 * Handler to set the value
+	 */
 	setValue?: (value: string) => void;
 	// Style
+
+	/**
+	 * Style of the text box. Same default as the "Input" component
+	 */
 	style?: InputStyle;
 	size?: TextAreaSize;
```

Comment:
This should have better documentation, like in DateInput, because it's not clear here what is TextAreaSize or how to have them